src/frontend/components/LiveVideoPanel/LiveVideoTabPanel/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Tab, Text } from 'grommet';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { LivePanelItem } from 'data/stores/useLivePanelState';
+import { ShouldNotHappen } from 'utils/errors/exception';
+import { theme } from 'utils/theme/theme';
+
+interface StyledTabProps {
+  selected?: boolean;
+}
+
+const colors = {
+  active: theme.global.colors['blue-active'],
```

Comment:
please order property in alphabetical order

Reply:
https://github.com/openfun/marsha/pull/1248/commits/a9b856d179526fac42707213d5fcd03b6d404348